packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -65,6 +95,18 @@ class Send extends Component {
       token
     } = this.props;
 
+    const recalculateMax = (args, state, { changeValue }) => {
```

Comment:
Can you put this as a class field?

Reason: perfomance. In this impl, we re-create a new function on each render